src/main/java/com/fundation/webservice/controller/Controller.java

```diff
@@ -860,6 +866,218 @@ else if (configJson.getString("metadata").equals("xmp")) {
         return new PPTtoPdfResponse(fileName, fileDownloadUri, outputChecksumString);
     }
 
+    /**
+     * Converts from a pdf document to an html page based on a user-provided criteria: input, config and output.
+     *
+     * @param asset The input pdf file itself
+     * @param input A JSON formatted string (wo \n characters) that holds all the parameters tied to the input image
```

Comment:
At the end of the comment, you missed the point "."

Reply:
Fixed.